p2p_wallet/Scenes/Main/Crypto/Components/Crypto Accounts/CryptoAccountsInteractor.swift

```diff
@@ -0,0 +1,132 @@
+import AnalyticsManager
+import Combine
+import Foundation
+import KeyAppBusiness
+import KeyAppKitCore
+import Resolver
+import Sell
+import SolanaSwift
+import SwiftyUserDefaults
+import Web3
+import Wormhole
+
+protocol CryptoAccountsInteractorProtocol {
```

Comment:
We agreed (when we didn't have you with us) naming protocol as `CryptoAccountsInteractor` and Implementation with suffix `Impl` (`CryptoAccountsInteractorImpl`). Let's follow this pattern